src/rgw/rgw_sal_motr.cc

```diff
@@ -1528,7 +1528,11 @@ int MotrObject::fetch_obj_entry_and_key(const DoutPrefixProvider* dpp, rgw_bucke
   if (rc < 0) {
       ldpp_dout(dpp, 0) <<__func__<< ": ERROR: failed to get object entry. rc=" << rc << dendl;
       return rc;
-    }
+  }
+  if(ent.is_delete_marker()){
```

Comment:
cosmetic change:
Please add a space in 'if' condition - line no 1532
`if(ent.is_delete_marker()) {`